quinn-proto/src/connection/streams/mod.rs

```diff
@@ -216,7 +216,7 @@ impl<'a> SendStream<'a> {
             .state
             .send
             .get_mut(&self.id)
-            .map(|s| s.get_or_insert_with(|| Box::new(Send::new(max_send_data))))
+            .map(get_or_insert_send(max_send_data))
```

Comment:
Was actually hoping for something that can also abstract over the `self.state.send.get_mut()` of it all.